src/table.rs

```diff
@@ -1254,4 +1267,24 @@ mod test {
 		});
 		assert_eq!(table.get(&key, 1, log.overlays()).unwrap(), None);
 	}
+
+	#[test]
+	fn multipart_collision() {
+		let dir = TempDir::new("multipart_collision");
+		let table = dir.table(Some(super::MAX_ENTRY_SIZE as u16), &rc_options());
+		let log = dir.log();
+
+		let key = key(1);
+		let val = value(32225); // This result in 0x7dff entry size, which conflicts with v4 multipart definition
+
```

Comment:
here when testing could also use
```
		let value_size = table.value_size();
		assert_eq!(0x7fdc, table.value_size()); // Max value size for this configuration.
		let val = value(value_size as usize); // This result in 0x7ffc entry size.
```
which is more than 32225, but I am not sure if the test try to touch the biggest value.

Reply:
The test is checking for 32225 specifically as it result in entry size `0x7dff`, which conflicted with the previous definition of MULTIHEAD (`0xfdff`)

Reply:
I've added an additional check in the test for max value size